requirements.txt

```diff
@@ -28,8 +28,12 @@ pylint==1.2.1
 pyparsing==2.0.3
 python-dateutil==2.4.2
 pytz==2015.4
+<<<<<<< HEAD
 requests==2.10.0
 runestone>=2.0.5
+=======
+runestone>=2.3.5
+>>>>>>> upstream/master
```

Comment:
Ooops, you checked in a version without resolving the conflicts.  I will fix this.